package.json

```diff
@@ -13,7 +13,8 @@
     "lint:prettier": "prettier --check .",
     "preinstall": "npx only-allow pnpm",
     "prepare": "lerna run build && husky install",
-    "publish:prerelease": "lerna publish --force-publish --canary major --dist-tag prerelease --ignore-scripts true --yes",
+    "publish:prerelease": "lerna publish --force-publish --canary minor --dist-tag prerelease --ignore-scripts true --yes",
```

Comment:
Unless they'll be called from outside a workflow, stick these two commands straight into the workflow files.